src/io/PipeReader.zig

```diff
@@ -399,6 +399,7 @@ pub fn WindowsPipeReader(
             }
             var this: *This = bun.cast(*This, fs.data);
             fs.deinit();
+            if (this.flags.is_done) return;
```

Comment:
is this an unrelated bug?

Reply:
unrelated mentioned in the PR also added a test for it just 1 line fix